src/waggle/data/vision.py

```diff
@@ -144,16 +144,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.capture.release()
 
     def snapshot(self):
-        timestamp = get_timestamp()
-        ok, data = self.capture.read()
+        ok = self.capture.read()
```

Comment:
I think you may want to use `grab` instead of `read` based on the OpenCV docs I found: https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#ae38c2a053d39d6b20c9c649e08ff0146

Reply:
Because of how Python's `not` operator works, the line below will actually work. :)

```python
# make up some fake return value tuple from read
ok = (False, b"some data")

# equals False
not ok
```

Reply:
Hello. That would've been a disaster. Thanks for finding that out. I tested the functions with grab and retrieve, but I submitted the code without updating the function name. Yes, it should've been grab, not read. Thanks for catching it up.